seacatauth/credentials/utils.py

```diff
@@ -0,0 +1,15 @@
+import asab.exceptions
+
+
+async def list_assigned_credential_ids(tenant_service, role_service, filter):
```

Comment:
As discussed on the review, this filter approach is not correct.
The flow needs to be split into two calls - one to retrieve info for UI, second "bulk apply" operates on the specific set of tenant ID (etc).